scib/metrics/morans_i.py

```diff
@@ -0,0 +1,149 @@
+import scanpy as sc
+import numpy as np
+import pandas as pd
+
+from scib.preprocessing import hvg_batch
+
+
+def morans_i(
+    adata_pre,
+    adata_post,
+    batch_key,
+    n_hvg=1000,
```

Comment:
1000 hvgs sounds like a lot for this as default. Is there any reason for this? Can it be reduced?

Reply:
I agree, this can be reduced - maybe to 100?